AmplifyPlugins/API/Podfile

```diff
@@ -1,7 +1,7 @@
 platform :ios, '11.0'
 use_frameworks!
 
-AWS_SDK_VERSION = "2.12.2"
+AWS_SDK_VERSION = "2.12.6"
```

Comment:
We're on .7 now

Reply:
will chat with @iartemiev offline about this, most places are 2.12.6, and a few like analytics is 2.12.2, so possibly there is a script that needs to be updated to catch all podfiles.